usaspending_api/search/v2/views/spending_by_category_views/spending_by_category.py

```diff
@@ -66,6 +67,18 @@ def post(self, request: Request) -> Response:
         original_filters = request.data.get("filters")
         validated_payload = TinyShield(models).block(request.data)
 
+        # Custom logic to support Subawards search for DEFC
+        # Check if it's the Spending by DEFC endpoint
+        if self.category.name == "defc":
```

Comment:
Here instead of checking if the whole set of original filters is None, we should be checking to see if "def_codes" exists as a filter on validated_payload.